lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -17,10 +18,20 @@ use if debug, 'Data::Dumper';
 
 sub base :Chained('/base') :PathPart('ia') :CaptureArgs(0) {
     my ( $self, $c ) = @_;
+
+    my $last_modified = $c->d->rs('InstantAnswer')->last_modified;
+    $last_modified->formatter( 'DateTime::Format::HTTP' );
+
+    $c->response->header (
+        'Last-Modified' => $last_modified;
+    );
```

Comment:
OK, I've added changes to the JSON route handlers which should set `Last-Modified` to max(updated) for index fetches and ia->updated for individual fetches.

They also short-circuit the request if it's for HEAD.